public/modules/dashboards/views/fbf.client.view.html

```diff
@@ -927,6 +938,25 @@ <h2 class="h5 modal-title">{{ "warning" | translate }}</h2>
     </div>
   </div>
 
+  <div id="printModal" tabindex="-1" class="modal" role="dialog">
```

Comment:
```suggestion
  <div id="printModal" tabindex="-1" class="modal hidden-print" role="dialog">
```
I think the "modal backdrop" is already invisible, but maybe it needs the explicit "background: transparent !important"-treatment...